src/pages/EnablePayments/IdologyQuestions.js

```diff
@@ -174,9 +176,10 @@ class IdologyQuestions extends React.Component {
                         message={errorMessage}
                         isLoading={this.props.walletAdditionalDetails.isLoading}
                         buttonText={this.props.translate('common.saveAndContinue')}
+                        containerStyles={[styles.mh0, styles.mv0, styles.mb0]}
```

Comment:
NAB, these style rules can be shifted to a class, as they're used on multiple pages